src/tree/LocalRootTreeItemBase.ts

```diff
@@ -367,6 +367,7 @@ export abstract class LocalRootTreeItemBase<TItem extends ILocalItem, TProperty
         if (!dockerInstallNotificationShownToUser && this._currentDockerStatus === 'NotInstalled') {
             dockerInstallNotificationShownToUser = true;
             // eslint-disable-next-line @typescript-eslint/no-floating-promises
```

Comment:
I'm not sure that this actually is active; you should be able to remove this top eslint disable.